modules/migration-state-migration-cli.adoc

```diff
@@ -18,3 +19,39 @@ You can perform a state migration between clusters or within the same cluster.
 ====
 State migration migrates only the components that constitute an application's state. If you want to migrate an entire namespace, use stage or cutover migration.
 ====
+
+.Prerequisites
+
+* The application on the source cluster has its state persisted in `PersistentVolumes` provisioned through `PersistentVolumeClaims`.
+
+* The application's manifests are available in a central repository that is accessible by both the source and the target clusters.
+
+
+.Procedure
+
+. Migrate persistent volume data from the source to the target cluster.
++
+You can perform this step as many times as needed. The source application continues running.
+
+. Quiesce the source application.
++
+You can do this by setting the replicas of workload resources to 0, either directly on the source cluster or by updating the manifests in GitHub before re-syncing the ArgoCD application.
```

Comment:
Because I'm not familiar with this topic area, I'm not sure from this line if the ArgoCD applications need to be re-synchronised only if the manifests are updated on GitHub, or if it is also needed if the replicas are set to 0 directly on the cluster. I'm assuming it's only required if the manifests are updated. Whichever it is, you could update this line to make it clear.